packages/compose/src/hooks/use-async-list/index.ts

```diff
@@ -30,26 +34,42 @@ function getFirstItemsPresentInState< T >( list: T[], state: T[] ): T[] {
  * React hook returns an array which items get asynchronously appended from a source array.
  * This behavior is useful if we want to render a list of items asynchronously for performance reasons.
  *
- * @param  list Source array.
+ * @param  list   Source array.
+ * @param  config Configuration object.
+ *
  * @return Async array.
  */
-function useAsyncList< T >( list: T[] ): T[] {
+function useAsyncList< T >(
+	list: T[],
+	config: AsyncListConfig = { step: 1 }
+): T[] {
+	const { step = 1 } = config;
```

Comment:
`step` has the default value assigned twice.

Reply:
I think it's not the same thing, the other default value is when config is undefined and this one is when "step" is undefined.

Reply:
In theory I can remove the "step" from the config default value but when I do so, typescript complains about the type.

Reply:
Ok, good 👍🏻